appveyor.yml

```diff
@@ -51,16 +51,20 @@ build_script:
         appveyor PushArtifact out/Default/symbols.zip
       }
 test_script:
-  - if "%GN_CONFIG%"=="testing" ( echo Verifying non proprietary ffmpeg & python electron\script\verify-ffmpeg.py --build-dir out\Default --source-root %cd% --ffmpeg-path out\ffmpeg )
   - ps: >-
-      if ($env:GN_CONFIG -eq 'testing') {
+      if ($env:GN_CONFIG -eq 'testing' -Or $env:GN_CONFIG -eq 'release') {
```

Comment:
This will run the tests for the publish / actual releasing builds which we don't want.